packages/appservice-rest/src/Utilities/KuduServiceUtility.ts

```diff
@@ -37,15 +37,16 @@ export class KuduServiceUtility {
         return deploymentID;
     }
 
-    public async deployUsingZipDeploy(packagePath: string): Promise<string> {
+    public async deployUsingZipDeploy(packagePath: string, customMessage?: any): Promise<string> {

```

Comment:
Any other Action where this lib method is used? Please validate those scenarios too, if this method is being used in any other scenario in this Action or any other Action

Reply:
I have verified this scenario